markdown_preview/webview_manager.py

```diff
@@ -9,12 +9,79 @@
 
 _ = init_gettext()
 
+SCRIPT_SCROLL_TO_CURSOR = """
```

Comment:
the script could be in a separate file, and loaded in the init. This way, it could be a js file with syntax coloration